src/interfaces/validator/validate-changes.js

```diff
@@ -63,7 +63,7 @@ export function validateChanges({incomingRecord, existingRecord, validate = true
   //debug(`db`);
   const normExistingRecord = normalizeRecord(existingRecord);
 
-  if (deepEqual(normIncomingRecord, normExistingRecord) === true) { // eslint-disable-line functional/no-conditional-statement
+  if (deepEqual(normIncomingRecord, normExistingRecord) === true) {
```

Comment:
https://github.com/chaijs/deep-eql recommends different style test. Is it possible to use it?
```
deepEqual(normIncomingRecord, normExistingRecord).should.be.true;
```

Reply:
Let's research this later when improving validate-changes otherwise.